src/init/inst-functions.sh

```diff
@@ -1117,6 +1117,19 @@ TransferShared()
     find ${INSTALLDIR}/etc/shared -maxdepth 1 -type f -not -name ar.conf -not -name files.yml -exec mv -f {} ${INSTALLDIR}/etc/shared/default \;
 }
 
+checkDownloadContent()
+{
+    VD_FILENAME='vd_1.0.0_vd_4.8.0.tar.xz'
+
+    if [ "${DOWNLOAD_CONTENT}" = "y" ] || [ "${DOWNLOAD_CONTENT}" = "yes" ]; then
```

Comment:
Aren't both conditions the same?

Reply:
If the variable comes from the package generation it is "yes", if it comes from preloaded-vars.conf in the installation by sources it is "y"

Reply:
You are right, thank you.